docs/source-pytorch/common/checkpointing_expert.rst

```diff
@@ -94,3 +98,37 @@ Custom Checkpoint IO Plugin
 .. note::
 
     Some ``TrainingTypePlugins`` like ``DeepSpeedStrategy`` do not support custom ``CheckpointIO`` as checkpointing logic is not modifiable.
+
+
+**************************
+Asynchronous Checkpointing
```

Comment:
Did you do any benchmarking?

Reply:
very basic one, just on BoringModel() trained on 1000 epochs. It was around 30% faster when I added some sleep time while saving the checkpoint. Do you have any recommendations?